### ISSUE TITLE
Recent update causing errors in admin panel

### ISSUE THREAD

**Issue by sarahbeverton** [getgrav/grav-plugin-admin#2357]
After updating to the most recent version of Grav and updating 5 plugins (including Admin Panel and Login), the Login plugin stopped being listed as a plugin in the Admin Panel. The website still works and we can log in to the Admin Panel (the Login plugin is still in the file system), but when we click 'Add' on the plugins page, we get an error. We updated the Admin Panel today to version 1.10.41.2 and still get the error. Here is a screenshot of the error when trying to add a plugin: 

![Screenshot 2023-05-12 at 3 41 02 PM](https://github.com/getgrav/grav-plugin-admin/assets/51892802/d7bc16b7-edd7-409b-8ad2-c92f78092f59)

**Comment by rhukster** [comment]
That all sounds pretty strange.  It might be that there was some error during the update procedure and the login plugin was corrupted?  Maybe backup/copy that login folder somewhere else and try a `bin/gpm install login` from the CLI to force install it again?

**Comment by rhukster** [comment]
I'm not realy seeing any issues. I just unziped the file you sent me, as the accounts folder was removed (correct move btw), I created a new accounts folder, was prompted to create my admin user, which I did, and logged in. Login i used during this process so that's a good sign.

I can list the plugins, login is clearly listed there. The one that is not seems to be a somewhat malformed custom plugin called `zenodo-data`.  I'm suprised this works at all as there's no data in the 'required' blueprints.yaml file.

Other than that, admin seems to work without issues, and i can even browse the site on the frontened.

During what actions exactly did you get that error above? I can't seem to replicate it on your site running locally.

**Comment by rhukster** [comment]
<img width="1566" alt="CleanShot 2023-05-15 at 09 46 29@2x" src="https://github.com/getgrav/grav-plugin-admin/assets/1084697/b80dba76-e3c4-4bdd-b2ac-822aa3f85e32">

**Comment by sarahbeverton** [comment]
Thanks for taking a look! That is interesting that you can see the login plugin. I still can't see it, and we also can't get it to appear when updating the login plugin on another website on our server. I get the error when I try to add a plugin (using the button on this screenshot). I assume you would not get the error since the error seems to have to do with the login plugin not appearing in the list.

![Screenshot 2023-05-15 at 11 54 01 AM](https://github.com/getgrav/grav-plugin-admin/assets/51892802/aae3ae03-b960-47f1-a66d-3796da3c5845)

**Comment by rhukster** [comment]
All I can think is that it might be related to permissions being messed up in the filesystem.  Do you have SSH access? can you login and look at the files directly?

**Comment by sarahbeverton** [comment]
Yes I can access the files - do you know what the permissions need to be?

**Comment by rhukster** [comment]
I would suggest checking the files ownership + permissions in the filesystem.  You will probably needs SSH access to do this, or at worst SFTP.

I didn't alter the files you sent me, so it the zip contained a fully working copy of Login plugin that works for me.. However, by  unzipping I set all the files to my local account and permissions, so I can't tell what you have on your end.

**Comment by sarahbeverton** [comment]
I finally figured out what was causing the problems (we were also having another problem with not being able to see or access accounts from the admin panel). There is a directory with the same name as our website under user/. It has a few .yaml files in it (email.yaml, flex-objects.yaml and security.yaml). When I changed the name of that directory to something other than our website, everything was fixed. Do you happen to know if that directory or those .yaml files are necessary, and if so, are they possibly configured incorrectly?

**Comment by rhukster** [comment]
it's not an issue, i think you are referring to the 'environments'.  Where as soon as you define an environment, and you are in it when using the admin, it will automatically save configuration there.  https://learn.getgrav.org/17/advanced/environment-config

Typically you should only use an environment for overrides, where you don't plan on using the admin... for example, on your live site to enable analytic scripts.

Then in a staging setup where you use admin, don't have an environment folder, so the configuration is stored into the default user/config folder.  Production will use any overrides it finds it the `yoursite.com` environment files, and fall back to the user/config based configuration for everything else.. that in turn falls back to the system/config folder as needed.

**Comment by PeopleInside** [comment]
Sorry but I cannot understood you.
I never touched server files, I only use the grav interface so.. today discovered I was unable to login to my profile.
The post of [sarahbeverton](https://github.com/sarahbeverton) https://github.com/getgrav/grav-plugin-admin/issues/2357#issuecomment-1552118847 is what really help to resolve the issue.

The Login plugin was not visible and the admin profile inaccessible.
I had to go on server files and rename the folder with the domain in the user folder, this resolve the issue.

Anyway thanks to Sara I resolved. I need maybe remember for the future how to do if Grav will let this happen again.
